lib/queryUtil.js

```diff
@@ -14,5 +14,17 @@ module.exports = {
             }
             return callback(null, result);
         });
+    },
+
+    exists: function(domain, query, callback) {
```

Comment:
A better question might be why does `count` return a boolean? Maybe just rename `count` to `exists`?